package.json

```diff
@@ -7,15 +7,16 @@
     "fs": false
   },
   "scripts": {
-    "test": "DEBUG=mfs:* aegir test -f test/browser.js",
+    "test": "cross-env DEBUG=mfs:* aegir test -f test/browser.js",
     "test:node": "aegir test -t node",
     "test:browser": "aegir test -t browser -t webworker -f test/browser.js",
     "build": "aegir build",
     "lint": "aegir lint",
     "release": "aegir release",
     "release-minor": "aegir release --type minor",
     "release-major": "aegir release --type major",
-    "coverage": "aegir coverage"
+    "coverage": "aegir coverage",
+    "commit-lint": "commitlint --extends=@commitlint/config-conventional --from=remotes/origin/master --to=HEAD"
```

Comment:
Let's not do this here, it'll come from aegir itself

Reply:
Can I remove it when it's in aegir instead?  Without this it's potluck as to whether Jenkins will quibble about the commit message or not.  It'd be a much better experience to know about it ahead of time.

Reply:
Fair enough, sounds reasonable.